### ISSUE TITLE
Smooch.open does not open conversation widget in FF 59.0.2 (TypeError)

### ISSUE THREAD

**Issue by kschiu** [smooch/smooch-web#664]
**I'm submitting a ...**
  - [x] bug report

 **What is the current behavior?**
* `Smooch.open()` does not properly open the conversation widget in Firefox 59.0.2
* Conversation widget opens properly in Chrome Version 66.0.3359.139

 **Please provide the steps to reproduce and, if possible, a minimal demo of the problem** 
* Invoke `Smooch.open` in Firefox 59.0.2

 **What is the expected behavior?**
* `Smooch.open()` opens the conversation widget in Firefox 59.0.2

 **Please tell us about your environment:**
- Delivery method: [ CDN ]
- Version [`Smooch.4.11.2`]
- Browser: [ Firefox 59.0.2  ]
- OS: [ OS X 10.13.1 ]

 **Other information**

Stacktrace:
![smooch_ff_error](https://user-images.githubusercontent.com/7429760/40149234-7176922a-5927-11e8-93d3-eae47a78faad.png)

Error seems to be in `frame.4.11.2` ?

Was able to get a little more information by catching all exceptions:
![smoochtypeerror](https://user-images.githubusercontent.com/7429760/40150993-f8da80dc-5931-11e8-863b-79124cbe2a64.png)

Apparently in FF, when `getComputedStyle()` is called inside an `iframe` with `display: none`, `null`  is returned: [link here](https://bugzilla.mozilla.org/show_bug.cgi?id=548397)

**Comment by alavers** [comment]
Thanks for the detailed report! The fix seems fairly straightforward.

**Comment by alavers** [comment]
Fixed in [4.11.3](https://github.com/smooch/smooch-web/releases/tag/4.11.3)

**Comment by kschiu** [comment]
@alavers 

It looks like we're still running into the issue in v4.11.3:
<img width="1046" alt="screen shot 2018-05-29 at 10 11 18 am" src="https://user-images.githubusercontent.com/7429760/40674139-cafad792-6328-11e8-9cf8-b83ed8ae7f55.png">

Is there any other information I can provide you with? Thanks!

**Comment by alavers** [comment]
@kschiu yes looking closer now and it does seem the fix we shipped is helpful, but incomplete.

Can you share with me the context in which you're running v4.11.3? I wasn't actually able to reproduce this bug with Firefox 59.0.2, but if I had a reliable repro I'd stand a better chance at fixing it.

**Comment by alavers** [comment]
@kschiu I'm asking about the page in which you're hosting the widget, since I'm unable to reproduce the issue myself.

In any case, I've implemented another fix that just went out with the [4.12.0 release](https://github.com/smooch/smooch-web/releases/tag/4.12.0). Could you re-test?
